frappe/core/doctype/communication/communication.py

```diff
@@ -118,7 +118,7 @@ def set_sender_full_name(self):
 						sender_name = None
 
 				self.sender = sender_email
-				self.sender_full_name = sender_name or get_fullname(frappe.session.user)
+				self.sender_full_name = sender_name #or get_fullname(frappe.session.user) #makes sender as administrator if not found
```

Comment:
`if not frappe.session.user=='Administrator'` ? sender_full_name is very nice to have in the view